<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pointPlacement</class>
 <widget class="QDialog" name="pointPlacement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point Placement Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Right-Click to place a point&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Point Sets&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="setCombo"/>
   </item>
   <item>
    <widget class="QTableWidget" name="setTable">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>61</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Z</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Color</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Label</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="newPointsButton">
     <property name="toolTip">
      <string>Press to create a new 3D Point Buffer Object</string>
     </property>
     <property name="text">
      <string>New Point Set</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="userInfo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="refreshButton">
     <property name="toolTip">
      <string>Press to refresh the list of Point Sets</string>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
